c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1661,34 +1611,104 @@ std::shared_ptr<void> getIpcDevPtr(std::string handle) {
   return sp;
 }
 
-void* raw_alloc(size_t nbytes) {
-  if (nbytes == 0) {
-    return nullptr;
+} // namespace Native
+
+// Define config stuff here, rather than its own .cpp file,
+// because parseArgs needs to know kLargeBuffer.
+bool parsed = false;
```

Comment:
Is this set anywhere?

Reply:
Hmm, maybe i did forget to set this, but im reworking this section anyway per https://github.com/pytorch/pytorch/pull/65365#discussion_r827309220.